sqlstore/src/it/java/uk/ac/bbsrc/tgac/miso/HibernatePaginatedDataSourceIT.java

```diff
@@ -22,9 +22,9 @@
 import uk.ac.bbsrc.tgac.miso.core.data.type.PlatformType;
 import uk.ac.bbsrc.tgac.miso.core.util.DateType;
 import uk.ac.bbsrc.tgac.miso.core.util.LimsUtils;
+import uk.ac.bbsrc.tgac.miso.core.util.PaginatedDataSource;
 import uk.ac.bbsrc.tgac.miso.core.util.PaginationFilter;
 import uk.ac.bbsrc.tgac.miso.core.util.TransferType;
-import uk.ac.bbsrc.tgac.miso.persistence.impl.HibernatePaginatedDataSource;
 
 public abstract class HibernatePaginatedDataSourceIT extends AbstractDAOTest {
```

Comment:
Worth renaming this class?

Reply:
Renamed `PaginationFilterSinkIT` since it's really integration testing the application of the methods from that `PaginationFilterSink`